gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/QueryAndGatewayDtoConverter.java

```diff
@@ -493,6 +493,10 @@ public static Filter addTimeSpaceAndIdFiltersAndConvertToQueryFilter(
       List<String> entityIdAttributes,
       org.hypertrace.gateway.service.v1.common.Filter providedFilter) {
 
+    if (entityIds.isEmpty()) {
```

Comment:
Do we still need this? Can we avoid the default instance filter?

Reply:
removed it